apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html

```diff
@@ -17,7 +17,7 @@
   </div>
 
   <div *slide id="angular-cli">
-    <h1 i18n>Angular CLI / Intro</h1>
+    <codelab-breadcrumb i18n>Intro</codelab-breadcrumb>
```

Comment:
Isn't it better to do something like <codelab-breadcrumb [data]="Intro"></codelab-breadcrumb>
Or there gonna be problems with i18n?

Reply:
It would prob be a more proper way, but ☝️ is way shorter